Applications/Terminal/main.cpp

```diff
@@ -124,15 +125,35 @@ int main(int argc, char** argv)
     slider->set_range(0, 100);
     slider->set_value(100);
 
+    auto* beep_choice_window = new GWindow;
+    beep_choice_window->set_title("Terminal beep settings");
+    beep_choice_window->set_rect(50, 50, 200, 100);
+
+    auto* radio_buttons = new GWidget;
+    beep_choice_window->set_main_widget(radio_buttons);
+    radio_buttons->set_fill_with_background_color(true);
+    radio_buttons->set_layout(make<GBoxLayout>(Orientation::Vertical));
+    radio_buttons->layout()->set_margins({ 4, 4, 4, 4 });
+
+    auto* sysbell_radio = new GRadioButton("Use (Audible) System Bell", radio_buttons);
+    auto* visbell_radio = new GRadioButton("Use (Visual) Terminal Bell", radio_buttons);
+    sysbell_radio->set_checked(terminal.should_beep());
+    sysbell_radio->on_checked = [&terminal] (const bool res) {
+                                    terminal.set_should_beep(res);
+                                };
+
     auto new_opacity = config->read_num_entry("Window", "Opacity", 255);
     terminal.set_opacity((float)new_opacity / 255.0);
 
     auto menubar = make<GMenuBar>();
 
     auto app_menu = make<GMenu>("Terminal");
     app_menu->add_action(GAction::create("Adjust opacity...", [opacity_adjustment_window] (const GAction&) {
-        opacity_adjustment_window->show();
-    }));
+                                                                  opacity_adjustment_window->show();
+                                                              }));
+    app_menu->add_action(GAction::create("Change audio output...", [beep_choice_window] (const GAction&) {
```

Comment:
`audio output` is a pretty strange name for this, how about `beep settings` so we are consistent with the window title? :)